index.src.html

```diff
@@ -1409,7 +1416,10 @@
 :: A {{VideoDecoderConfig}} that authors may use to decode the associated
     {{EncodedVideoChunk}}.
 
-: <dfn dict-member for=EncodedVideoChunkMetadata>temporalLayerId</dfn>
+: <dfn dict-member for=EncodedVideoChunkMetadata>svc</dfn>
+:: A collection of metadata related to the {{EncodedVideoChunk}}'s SVC options.
```

Comment:
my attempt: 

A collection of metadata describing this chunk with respect to the configured scalabilityMode.

Reply:
Seems good. Done. Thanks!